### DISCUSSION TITLE
Need help modifying completion history displayed lines

### DISCUSSION THREAD

**Discussion by justinesmithies** [akinomyoga/ble.sh#189] [General]
Maybe a daft question but I can't see the answer for looking. How do I stop the last line from being displayed so if i use the completion history that line is not displayed , I just get to cycle through the history ???

```
╭─justine@justine-laptop ~ took 20s 
╰> config status
(nsearch#2: << !497   ) `config '

**Comment by akinomyoga** [comment]
The detailed setup depends on how you bind or enable the non-incremental history search, but it can be turned off using `hide-status` described in [§4.1.0.1](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#4101-widget-history-search). If you set up the non-incremental search by `~/.inputrc` or `bind`, you need to switch to `ble-bind`. E.g.

```
# blerc
ble-bind -f prior 'history-search-backward hide-status:empty=emulate-readline'
```
